components/molecules/dropdown/components/item.js

```diff
@@ -27,18 +27,62 @@ const renderItem = ({ id, name, to }, closeDropdown, onClick) => {
   )
 }
 
-export const DropdownItem = ({ closeDropdown, item, onClick }) => {
-  return (
-    <StyledDropdownItem divider={item.divider}>
-      {item.divider ? <StyledDivider /> : renderItem(item, closeDropdown, onClick)}
-    </StyledDropdownItem>
-  )
+export const DropdownItem = ({ closeDropdown, elementType, item, onClick }) => {
+  switch (elementType) {
+    case elementTypes.List:
+      return (
+        <StyledDropdownItem divider={item.divider}>
+          {item.divider ? <StyledDivider /> : renderItem(item, closeDropdown, onClick)}
+        </StyledDropdownItem>
+      )
+    case elementTypes.Colour:
+      return (
+        <StyledColourItem
+          colour={item.colour}
+          onClick={() => {
+            onClick(item)
+          }}
+        />
+      )
+    default:
+      return 'invalid elementType'
+  }
 }
+const StyledColourItem = styled.div`
+  width: 1.55rem;
+  height: 1.55rem;
+  ${({ colour }) =>
+    css`
+      background-color: ${colour};
+    `}
+  ${({ colour }) => {
+    const size = '0.5rem'
```

Comment:
@mhsnprvr2 We have theme variables - look at other packages - size should either be a prop of the compoennt or a theme variable so we can possibly change - this bit is a little ugly, very rarely have we had to add conditionals into the styled components sections